.github/workflows/molecule.yml

```diff
@@ -41,12 +41,15 @@ jobs:
           mkdir -p ~/.ansible/collections/ansible_collections/$COLLECTION_NAMESPACE
           cp -a ../ansible-collection-$COLLECTION_NAME ~/.ansible/collections/ansible_collections/$COLLECTION_NAMESPACE/$COLLECTION_NAME
 
+      - name: Install geerlingguy.mysql roles
+        run: ansible-galaxy install geerlingguy.mysql
```

Comment:
Another way to have that is the `prepare.yml` file. But it's totally fine to have it here. Having it in `prepare.yml` will help with local tests, though.

Reply:
When we set this task in prepare only, there was error. Molecule could not find the role. I can prepare once again